djangocms_blog/settings.py

```diff
@@ -147,5 +147,25 @@ def get_setting(name):
         'BLOG_PLUGIN_TEMPLATE_FOLDERS': getattr(
             settings, 'BLOG_PLUGIN_TEMPLATE_FOLDERS', (('plugins', _('Default template')),)),
 
+        'BLOG_LINKED_IMAGE': (
```

Comment:
Do we really need this?
wouldn't be easier for the user to upload the cover image by using the existing fields?
I suspect the rules below only applies to youtube and they are not generic enough to justify the added logic